dev/io.openliberty.checkpoint_fat/test-applications/mphealth/resources/WEB-INF/web.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE web-app PUBLIC "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN" "http://java.sun.com/dtd/web-app_2_3.dtd">
+<web-app id="WebApp_ID" version="3.0" xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
+
```

Comment:
I would prefer we use servlet annotations instead of `web.xml` for our tests.  It makes it easier to follow.
```java
@WebServlet(name = "HealthServlet", "urlPatterns = "/mphealth", loadOnStartup = 1)
```

Reply:
Added the annotation instead of web.xml